### ISSUE TITLE
“setLocality” Not implemented while using abrmd with “--tcti=socket”

### ISSUE THREAD

**Issue by leeehui** [tpm2-software/tpm2-abrmd#359]
In my opinion, the following error should only occur when using abrmd with “--tcti=device”, 
because "setLocaliy" function in tpm2-tss has an implementation with socket tcti.

[SocketSetLocality](https://github.com/tpm2-software/tpm2-tss/blob/c560204d4b0151776cad75f4f812a76206d3c120/tcti/tcti_socket.c#L235-L254)

but, there is still an error showing setlocality is not supported,  

GDBus.Error:com.intel.tss2.Tabrmd.Error.NotImplemented: setLocality function not implemented.

I know nothing about DBus, so I wonder if the above analysis is right.
thanks.

**Comment by flihp** [comment]
You're right that it's possible to cancel commands if the TCTI that the tabrmd is using on the backend supports the command. Someone has to write the code to get the cancel command from the client, down through the daemon though and that's non-trivial. Would love to review a patch adding the feature.

**Comment by leeehui** [comment]
I’m confused about your last comment. Did you mean, if the abrmd receives a setlocality request, it may cancel the tpm command that is currently being executed?
